SkillChat.Client.ViewModel/MainWindowViewModel.cs

```diff
@@ -479,8 +479,16 @@ public MainWindowViewModel()
 
                     settings.AccessToken = Tokens.AccessToken;
                     settings.RefreshToken = Tokens.RefreshToken;
-                    settings.UserName = RegisterUser.Login;
+                    settings.UserName = RegisterUser.UserName;
+                    settings.Login = RegisterUser.Login;
                     configuration.GetSection("ChatClientSettings").Set(settings);
+
+                    //Очистка полей регистрации
+                    RegisterUser.Login = "";
+                    RegisterUser.Password = "";
+                    RegisterUser.UserName = "";
+                    RegisterUser.Consent = false;
```

Comment:
Лучше вынести в метод Clear для RegisterUser
```suggestion
                    //Очистка полей регистрации
                    RegisterUser.Clear();
```

Reply:
> Нужно ещё чуть дошлифовать и будет ок

Сделано